scripts/doc_templates/templates/protex

```diff
@@ -296,8 +296,8 @@ LINE:
 
 #       Straight quote
 #       --------------
-        if ($Fld[1] eq '!QUOTE:') {
-           for ($i = 2; $i <= $#Fld; $i++) {
+        if ($Fld[0] eq '!QUOTE:') {
+           for ($i = 1; $i <= $#Fld - 1; $i++) {
```

Comment:
I am removing the change you had here (subtracting 1 from `$#Fld`): `$#Fld` gives the index of the last element, and I think the correct thing to do is still to loop until `i` is equal to the index of the last element. (I did a little test and `$#Fld` changes with `$[`, which I think makes it correct to *not* subtract 1 here.)